packages/bmx6-auto-gw-mode/files/etc/watchping/wan6-fail.d/bmx6-gw

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+logger -t bmx6-auto-gw "Searching IPv6 Internet access into the mesh"
+bmx6 -c tunIn -inet6
+bmx6 -c tunOut inet6 /network 2000::/3 /maxPrefixLen 96
```

Comment:
On the current generic LiMe configuration, there are two default tunnel announcements named "inet6" and "publicv6".
The first (inet6) refers to an internet Uplink (maxPrefixLen=0), the second to any global network announced on the mesh with a maximum prefix of 64bits (this is according to RIPE rules).

> bmx6.inet6=tunOut
> bmx6.inet6.tunOut='inet6'
> bmx6.inet6.network='::/0'
> bmx6.inet6.maxPrefixLen='0'
> bmx6.publicv6=tunOut
> bmx6.publicv6.tunOut='publicv6'
> bmx6.publicv6.network='2000::/3'
> bmx6.publicv6.maxPrefixLen='64'

So bmx6-auto-gw should use _/network ::/0_ and _/maxPrefixLen 0_